pydevices/acq400Devices/acq1001_st.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```

Comment:
The previous is 'more correct'. The interpreter does not necessarily live a /usr/bin/python